src/kvstore/raftex/RaftPart.cpp

```diff
@@ -758,6 +758,13 @@ void RaftPart::replicateLogs(folly::EventBase* eb,
             break;
         }
 
+        if (term_ != currTerm) {
+            VLOG(2) << idStr_ << "Term has been updated, previous "
+                    << currTerm << ", current " << term_;
+            currTerm = term_;
+            break;
```

Comment:
"break" is right? You skip the line 768.

Reply:
My bad, good catch!